pages/index.vue

```diff
@@ -1,10 +1,13 @@
 <template>
-  <center>
-    <div>
-      <h2>This Week's Featured Tutorial</h2>
-      <h3>Tutorial Name - Username</h3>
-    </div>
-  </center>
+  <div class="container">
+    <Navbar />
```

Comment:
Why only in index.vue? Why not on every page?

Reply:
done